deploy/s3.tf

```diff
@@ -47,4 +47,8 @@ resource "aws_s3_bucket_policy" "ipni_store_policy" {
 
 resource "aws_s3_bucket" "notifier_head_bucket" {
   bucket = "${terraform.workspace}-${var.app}-notifier-head-bucket"
-}
\ No newline at end of file
+}
+
+resource "aws_s3_bucket" "claim_store_bucket" {
```

Comment:
Question: in prod, shouldn't this be the same bucket we already have to legacy claims? Or do we store it different.